src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private Long id;
```

Comment:
UserRequestDto is for registering new users, you don't need to specify id, GeneratedValue.Identity will do that for you